src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -118,400 +117,21 @@ public void start(String[] args) {
         // The preferences return the system newline character sequence as default
         Globals.NEWLINE = Globals.prefs.get(JabRefPreferences.NEWLINE);
 
-        Optional<Vector<ParserResult>> loaded = processArguments(args, true);
+        // Process arguments
+        ArgumentProcessor argumentProcessor = new ArgumentProcessor(args, true);
```

Comment:
Instead of passing booleans to constructors, it is better to use an enum instead. Because I have no idea what the true in that case means.